WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -469,6 +469,12 @@ protected void onCreate(Bundle savedInstanceState) {
                 }
             }
 
+            if (isRestarting && extras.getBoolean(EXTRA_IS_NEW_POST)) {
```

Comment:
> My guess is that the reason is that only overwriting mIsNewPost if the extra is true is a less invasive change compared to overwriting mIsNewPost any time extras.containsKey(EXTRA_IS_NEW_POST).

Exactly. At this point, the usage of `mIsNewPost` has become quite tricky to follow, so I wasn't sure how safe would it be to also introduce setting the flag to `false` at this code site. In the case this PR is trying to address, the flag is false because it still has its initialization value. The fix is to "turn it to true". But, can't be sure if there are other flows passing through this code site that would have the value as true while the extra is false. EditPostActivity is still a beast, so many flows :(